README.md

```diff
@@ -162,6 +162,7 @@ Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde bur
 
 ##### UI 
 - [Animated Selection Slide](https://github.com/sbilketay/animated_selection_slide) - Ana widget'ı kaydırarak sonrasında belirli hareketler ile bir işlem yapmanıza yardımcı olan kütüphane - Sezgin Bilgetay
+- [Flutter Music Player](https://github.com/sbilketay/flutter_music_player) Music Player UI - Sezgin Bilgetay
```

Comment:
Selamlar, tekrardan tesekkurler katkin icin!

Bu bir kutuphane mi yoksa bir egitici icerik projesi mi acaba?

Reply:
Kutuphane degil, suan sadece player animasyonlara sahip bir ui, gelistirmeye devam ediyorum. Yakin zamanda stream ile parca calma, playlist gibi ozellikler ekleyecegim.

Reply:
Anladim. O zaman ben bunu merge edip acik kaynak projeler diye bir kisim olusturup bunu oraya ekleyecegim.

Reply:
Süper. Teşekkürler.

Başka arkadaşlar da paylaşsa projelerini de incelesek. Çok güzel olur.

Reply:
Paylasırken arkadaşları projelerini eklemeleri için cesaretlendiririz eminim güzel şeyler gelecektir :)